tasks/test_syntax.js

```diff
@@ -260,12 +260,9 @@ function assertCircularDeps() {
         var circularDeps = res.circular();
         var logs = [];
 
-        // see https://github.com/plotly/plotly.js/milestone/9
-        var MAX_ALLOWED_CIRCULAR_DEPS = 16;
-
-        if(circularDeps.length > MAX_ALLOWED_CIRCULAR_DEPS) {
+        if(circularDeps.length) {
             console.log(circularDeps.join('\n'));
-            logs.push('some new circular dependencies were added to src/');
+            logs.push('some circular dependencies were found to src/');
```

Comment:
found *in* src/

Reply:
Good :eye: done in https://github.com/plotly/plotly.js/pull/2429/commits/d27b3baa880b8b41fd61d9caf3050d26ce7a7f19